src/modules/header/components/navigation/navigation.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from "classnames";
+import { Drawer } from "antd";
+import {
+    ApartmentOutlined,
+    AuditOutlined,
+    CloseOutlined,
+    ContactsOutlined,
+    FolderOutlined,
+    GlobalOutlined,
+    HomeFilled,
+    RiseOutlined,
+    UserOutlined,
+} from "@ant-design/icons";
+import styles from "./navigation.module.css";
+
+interface NavigationTypes {
+    isDrawerOpen: boolean;
+    setDrawerOpen: (arg1: boolean) => void;
+}
+
+const Navigation = ({ isDrawerOpen, setDrawerOpen }: NavigationTypes) => {
+    const onClose = () => {
+        setDrawerOpen(false);
+    };
+
+    return (
+        <Drawer
+            placement='left'
+            closable={false}
+            open={isDrawerOpen}
+            width={250}
+            styles={{
+                header: { padding: "10px", backgroundColor: "#2E2E2E" },
+                body: { padding: 0 },
+            }}
+        >
+            <div className={styles.nav_header}>
+                <CloseOutlined onClick={onClose} className={styles.close_icon} />
+            </div>
+            <div className={styles.drawerContent}>
+                <div className={classNames(styles.menu_item, styles.border_bottom, styles.active)}>
```

Comment:
i think bad idea to use div for links

Reply:
I haven't yet had the access to router at that time, so Links without routes might give errors, routing in navigation menu will be implemented in DEV-12/connect-sidebar-navigation-with-the-router issue